contrib/dnsmasq/docker0.conf

```diff
@@ -11,8 +11,22 @@ dhcp-host=52:54:00:d7:99:c7,172.17.0.24,1h
 enable-tftp
 tftp-root=/var/lib/tftpboot
 
+# Legacy PXE
+dhcp-match=set:bios,option:client-arch,0
+dhcp-boot=tag:bios,undionly.kpxe
+
+# UEFI
+dhcp-match=set:efi32,option:client-arch,6
+dhcp-boot=tag:efi32,ipxe.efi
+
+dhcp-match=set:efibc,option:client-arch,7
+dhcp-boot=tag:efibc,ipxe.efi
+
+dhcp-match=set:efi64,option:client-arch,9
+dhcp-boot=tag:efi64,ipxe.efi
+
+# iPXE
 dhcp-userclass=set:ipxe,iPXE
-dhcp-boot=tag:#ipxe,undionly.kpxe
 dhcp-boot=tag:ipxe,http://matchbox.foo:8080/boot.ipxe
```

Comment:
s/matchbox.foo/matchbox.example.com/

Reply:
I believe I originally generated the fake certs with `matchbox.foo` as the name, and these confs are actually used for the locally runnable examples.

Reply:
Actually nope. I believe we can safely change this now.

```
    X509v3 Subject Alternative Name: 
                DNS:matchbox.example.com, DNS:matchbox.foo
```

Reply:
Oops. This rename broke all the cluster examples. Fix in #676